tests/testUtils.py

```diff
@@ -50,6 +51,65 @@ class BlockLogAction(EnumType):
 addEnum(BlockLogAction, "prune_transactions")
 
 ###########################################################################################
+
+class Timeout:
```

Comment:
It's not timeout anymore, I would change the name to WaitSpec. Have an const object called defaultWaitSpec, use defaultWaitSpec for the default value of  functions  that require it.

Reply:
Changed to WaitSpec